### ISSUE TITLE
使用MySQL时alist admin创建的表名不对

### ISSUE THREAD

**Issue by mengguyi** [alist-org/alist#2842]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

v3.7.1

### Driver used / 使用的存储驱动

本地存储

### Describe the bug / 问题描述

在使用MySQL时alist admin创建的表名不对
这会导致后续索引不可用
![image](https://user-images.githubusercontent.com/73788063/210047437-ab466be1-d78a-452c-a77f-83cb957761c4.png)
`x_search_node_my_sqls`应为`x_search_nodes`
该问题只出现在v3.7.1使用MySQL时，使用sqlite3没有问题

### Reproduction / 复现链接

```
{
  "force": false,
  "address": "0.0.0.0",
  "port": 5244,
  "site_url": "",
  "cdn": "",
  "jwt_secret": "v3qQMj7qifPPXBBR",
  "token_expires_in": 48,
  "database": {
    "type": "mysql",
    "host": "127.0.0.1",
    "port": 3306,
    "user": "root",
    "password": "passwd",
    "name": "alist",
    "db_file": "",
    "table_prefix": "x_",
    "ssl_mode": ""
  },
  "scheme": {
    "https": false,
    "cert_file": "",
    "key_file": ""
  },
  "temp_dir": "data\\temp",
  "bleve_dir": "data\\bleve",
  "log": {
    "enable": true,
    "name": "data\\log\\log.log",
    "max_size": 10,
    "max_backups": 5,
    "max_age": 28,
    "compress": false
  },
  "max_connections": 0
}
```
按照如下配置执行`alist admin`

### Logs / 日志

[33mWARN[0m[2022-12-30 15:47:23] not enable search                            
[33mWARN[0m[2022-12-30 15:51:24] not enable search

**Comment by mengguyi** [comment]
在添加本地存储后
![image](https://user-images.githubusercontent.com/73788063/210048196-6b727011-b22b-4699-9243-be366557ae34.png)
启用索引
![image](https://user-images.githubusercontent.com/73788063/210048190-9fb018d9-9c78-460b-b184-c2d7d99b17a1.png)
控制台输出
![image](https://user-images.githubusercontent.com/73788063/210048181-c38407ff-75f4-47bf-a03c-98066d783b05.png)

**Comment by Xhofe** [comment]
fixed in 143e4cd0770dc4108eec8b47d0f0efc117dc85d5
